client/my-sites/site-settings/jetpack-sync-panel/index.jsx

```diff
@@ -171,12 +171,12 @@ class JetpackSyncPanel extends React.Component {
 				<div className="jetpack-sync-panel__action">
 					{ translate(
 						'Jetpack Sync keeps your WordPress.com dashboard up to date. ' +
-							'Data is sent from your site to the WordPress.com dashboard regularly to provide a faster experience. '
+							'Data is sent from your site to the WordPress.com dashboard regularly to provide a faster experience.'
 					) }
 
 					{ ! this.shouldDisableSync() &&
 						translate(
-							'If you suspect some data is missing, you can {{link}}initiate a sync manually{{/link}}.',
+							' If you suspect some data is missing, you can {{link}}initiate a sync manually{{/link}}.',
```

Comment:
:ok: This change is safe, we'll keep the existing 20 translations.

Reply:
I think you should add the space outside the `translate` call, otherwise translators are likely to miss that space.